schedule/yast/lvm/lvm_thin_provisioning_x86_64.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name: lvm_thin_provisioning
+description: >
+  Complete OS deployment with unencrypted LVM drive management. Test creates
+  2 LVM and BIOS boot partitions. Thin pool and thin lv reside on the second
+  LVM partition, where /home (XFS) is being mounted. Partitioning is validated
+  in the booted system after the installation.
+vars:
+  YUI_REST_API: 1
+schedule:
+  extension_module_selection:
+    - installation/module_registration/register_module_desktop
+  add_on_product:
+    - installation/add_on_product/skip_install_addons
```

Comment:
In the adaption in [PR#16218](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16218) this will be the default already.